Source/WebCore/editing/cocoa/AttributedString.mm

```diff
@@ -183,7 +186,7 @@
     if ([value isKindOfClass:WebCore::getNSTextAttachmentClass()])
         return { { { RetainPtr { (NSTextAttachment *)value } } } };
     if ([value isKindOfClass:PAL::getUIFontClass()])
-        return { { { RetainPtr { (UIFont *)value } } } };
+        return { { { Font::create(FontPlatformData((CTFontRef)value, [(UIFont *)value pointSize])) } } };
```

Comment:
The whole point of `PAL::getUIFontClass()` is so you don't cast to `UIFont` directly.